### DISCUSSION TITLE
swagger-ui

### DISCUSSION THREAD

**Discussion by skrick** [PostgREST/postgrest#2090] [Q&A] [answered]
What is the "best" way to configure swagger to work with postgrest. I have tried to use docker to link my postgrest and swagger-ui but I cannot figure out how to enable the Bearer security auth. All of my API calls require a valid jwt token.

**Comment by wolfgangwalther** [comment, marked as answer]
I created an issue in the docs repo. Maybe we can find somebody to provide a "how to" on how to do this the best way.
